client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/persistence/KaaClientPropertiesState.java

```diff
@@ -257,6 +258,13 @@ public void persist() {
     }
 
     @Override
+    public String refreshEndpointAccessToken() {
+        String newAccessToken = UUID.randomUUID().toString();
+        setEndpointAccessToken(newAccessToken);
+        return newAccessToken;
```

Comment:
Maybe it is redundant, but I suggest to get the access token the same way as we set it.